pkg/apis/awsprovider/v1alpha1/awsclusterproviderconfig_types.go

```diff
@@ -53,6 +54,18 @@ type AWSClusterProviderSpec struct {
 
 	// SAKeyPair is the service account key pair.
 	SAKeyPair KeyPair `json:"saKeyPair,omitempty"`
+
+	// InitConfiguration is used to customize any kubeadm init configuration
+	// parameters.
+	// Cannot override certain variables including:
+	// * noderegistration.Name
+	// * noderegistration.criSocket
+	// * apiserver.ExtraArgs[cloud-provider] = aws
+	InitConfiguration v1beta1.InitConfiguration `json:"initConfiguration,omitempty"`
```

Comment:
No ClusterConfiguration? It seems like the ability to embed ClusterConfiguration in the InitConfiguration is not coming through in the generated CRD.

Reply:
ah yeah, it is totally missing, was hoping it'd pick it up but I guess embedded types are trickier